build.gradle

```diff
@@ -47,6 +47,13 @@ minecraft {
             property 'mixin.env.remapRefMap', 'true'
             property 'mixin.env.refMapRemappingFile', "${buildDir}/createSrgToMcp/output.srg"
         }
+
+        clientRandom {
+            parent runs.client
+            // #### will be replaced with a random number | UUID is a randomly generated one
+            args '--username', 'Dev####', '--uuid', 'fdb0e318-1ac3-4049-b6f8-d024c0b87a56'
```

Comment:
Now when you open another game window and connect to the server, there will be no "same nicknames" error?

Reply:
If you start clientRandom you will be assigned a random number meaning it avoids the issue of duplicate names
Multiple instances of the same clientRandom will have the same uuid though (can affect some things, e.g. the skin)